prime-router/src/main/kotlin/azure/SettingsFacade.kt

```diff
@@ -60,13 +61,23 @@ class SettingsFacade(
     }
 
     override fun findReceiver(fullName: String): Receiver? {
-        val pair = Receiver.parseFullName(fullName)
-        return findSetting(pair.second, ReceiverAPI::class.java, pair.first)
+        try {
+            val pair = Receiver.parseFullName(fullName)
+            return findSetting(pair.second, ReceiverAPI::class.java, pair.first)
+        } catch (e: Exception) {
+            logger.warn("Cannot find receiver: ${e.localizedMessage} ${e.stackTraceToString()}")
+            return null
```

Comment:
Can you explain what this null value will do in the rest of the code?

Reply:
The null  is already implicit on the return value for `findSettings`. It currently happens for invalid users that have between 1-2 dot-separated segments.

lint demanded it be explicitly added here.